cycledash/static/scss/examine.scss

```diff
@@ -549,7 +584,7 @@ a.download-vcf {
 }
 
 #pileup-container .controls {
-  flex: 0 0 35px;  /* fixed height */
+  flex: 0 0 38px;  /* fixed height */
```

Comment:
Yup! But I actually don't need that much height anymore. Gonna push a tiny tweak.